hud_teaminfo.c

```diff
@@ -239,16 +246,21 @@ qbool Has_Both_RL_and_LG(int flags)
 	return (flags & IT_ROCKET_LAUNCHER) && (flags & IT_LIGHTNING);
 }
 
-static int SCR_HudDrawTeamInfoPlayer(ti_player_t *ti_cl, float x, int y, int maxname, int maxloc, qbool width_only, float scale, const char* layout, int weapon_style, int armor_style, int powerup_style, int flag_style, int low_health, qbool proportional)
+static int SCR_HudDrawTeamInfoPlayer(ti_player_t *ti_cl, float x, int y, int maxname, int maxloc, qbool width_only, float scale, const char* layout, int weapon_style, int show_ammo, int show_countdown, int armor_style, int powerup_style, int flag_style, int low_health, qbool proportional)
 {
 	extern cvar_t tp_name_rlg;
-
-	char *s, *loc, tmp[1024], tmp2[MAX_MACRO_STRING], *aclr;
+	char *s, *loc, tmp[1024], tmp2[MAX_MACRO_STRING], *aclr, *txtclr;
+	char * ktxmode = Info_ValueForKey(cl.serverinfo, "mode"); // check KTX game mode
 	float x_in = x; // save x
-	int i;
+	int i, a;
+	qbool isDeadCA, isRespawning;
+	qbool ktx_ca = (strstr(ktxmode, "-ca") != NULL);
+	qbool ktx_wipeout = (strstr(ktxmode, "-wo") != NULL);
+	qbool ktx_ca_wipeout = (ktx_ca || ktx_wipeout);
```

Comment:
Hey dusty!
Should we not have a little helper function for this? It is also in cl_parsers.c.